lib/start.js

```diff
@@ -134,14 +138,29 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
   if (options.network_log) {
     let exitCode = 0
     let jsonOutput = {}
+    // If the log is still being written, give it time to finish (10 seconds)
```

Comment:
seems like we should just retry every second until we can read it?

Reply:
10 seconds seems excessive

Reply:
I can definitely take the sleep out; when I was testing this, I wasn't sure if the file handle was closed when the read is taking place. It does seem that is the case though! The output in the log is just not proper JSON

Reply:
while you're at it, can you move all this logic into a method/another file?

Reply:
Done